packages/frontend/src/components/create-campaign/preview/index.tsx

```diff
@@ -55,6 +61,72 @@ export function CampaignPreview({
     const publicClient = usePublicClient();
     const { writeContractAsync } = useWriteContract();
 
+    const tokensToApprove = useMemo(() => {
+        if (payload.rewardType === RewardType.tokens) return payload.tokens;
+
+        if (payload.rewardType === RewardType.points && payload.feeToken) {
+            const { amount, token } = payload.feeToken;
+
+            const newRaw = amount.raw + (amount.raw * 15n) / 100n;
```

Comment:
Slightly more efficient: amount.raw * 115n / 100

Reply:
Addressed in b2e893253bb59bfceac8e3195fa15f23d58512e3.